multi-agent/adapters/outbound/mongo/builtin_user_config_repository.py

```
@@ -0,0 +1,73 @@
+from typing import List, Optional
+
+import pymongo
+
+from mas.resources.builtin_models import BuiltinUserConfig
+from mas.resources.repository.builtin_user_config_repository import (
+    BuiltinUserConfigRepository as BuiltinUserConfigRepositoryPort,
+)
+
+
+class MongoBuiltinUserConfigRepository(BuiltinUserConfigRepositoryPort):
+    def __init__(
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)